### DISCUSSION TITLE
Release an updated version of 0.13

### DISCUSSION THREAD

**Discussion by levino** [withastro/starlight#1271] [Feature Requests]
### What version of `starlight` are you using?

0.13.1

### What is your idea?

I have [patched](https://github.com/levino/starlight/pull/1) version 0.13 and would like you to release a new version 0.13.2 with the patches. To this end one probably has to maintain a branch `v0.13` where these patches are managed and from which the publication of the patched versions happen.

### Why is this feature necessary?

As [explained](https://github.com/withastro/starlight/pull/1270), `starlight-blog` does not work with `starlight@>=0.14` and it does not look like this problem will be fixed right away. I have tested `starlight@0.13.1` with `starlight-blog@0.4.0` and `astro@4` and [it seems to work](https://github.com/HiDeoo/starlight-blog/pull/21). However `pnpm` complains that `starlight@0.13.1` has the unmet peer dep `astro@^3` and I use `astro@4` instead (and some sub dependencies). This release would silence that warning and restore order.

### Participation

- [x] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Sorry for the roadbumps! Given Starlight is still beta software, we probably don’t have the bandwidth to maintain older branches. Instead, hopefully `starlight-blog` will be able to offer full compatibility with the latest versions soon. I’ll aim to help @HiDeoo get #1175 over the line to move this along!

**Reply by levino** [reply]
I think the effort to "maintain" a branch `v0.13` is very low and the annoyance for new users is harming the project. Taking a couple of minutes to branch out, merge and release a patched version is well spent time. But it is your call. Good luck with fixing the other issue soon. I will come back to this in the new year, if the issues still persists.
